.github/workflows/regression-tests.yml

```diff
@@ -0,0 +1,120 @@
+      zkevm_dac_commit_id:
+        description: 0xPolygon/cdk-data-availability (commit id)
+        required: true
+      zkevm_node_commit_id:
+        description: 0xPolygon/cdk-validium-node (commit id)
+        required: true
+      bake_time:
+        description: bake time (minutes)
+        required: false
+        default: 30
+jobs:
+  deploy_devnet:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: '${{ github.event.before }}'
+      - name: Set up Docker
+        uses: docker/setup-buildx-action@v1
+      - name: Clone and build agglayer
+        run: |
+          git clone https://github.com/0xPolygon/agglayer.git
+          cd agglayer
+          git checkout "${{ github.event.inputs.zkevm_agglayer_commit_id }}"
+          docker compose -f docker/docker-compose.yaml build --no-cache agglayer
+      - name: Clone and build zkevm-bridge-service
+        run: |
+          git clone https://github.com/0xPolygonHermez/zkevm-bridge-service.git
+          cd zkevm-bridge-service
+          git checkout "${{ github.event.inputs.zkevm_bridge_service_commit_id
+          }}"
+          docker build -t zkevm-bridge-service:local -f ./Dockerfile .
+      - name: Clone and build zkevm-bridge-ui
+        run: |
+          git clone https://github.com/0xPolygonHermez/zkevm-bridge-ui.git
+          cd zkevm-bridge-ui
+          git checkout "${{ github.event.inputs.zkevm_bridge_ui_commit_id }}"
+          docker build -t zkevm-bridge-ui:local -f ./Dockerfile .
+      - name: Clone and build cdk-data-availability
+        run: |
+          git clone https://github.com/0xPolygon/cdk-data-availability.git
+          cd cdk-data-availability
+          git checkout "${{ github.event.inputs.zkevm_dac_commit_id }}"
+          docker build -t cdk-data-availability:local -f ./Dockerfile .    
+      - name: Clone and build cdk-validium-node
+        run: |
+          git clone https://github.com/0xPolygon/cdk-validium-node.git
+          cd cdk-validium-node
+          git checkout "${{ github.event.inputs.zkevm_node_commit_id }}"
+          docker build -t cdk-validium-node:local -f ./Dockerfile .
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+      - name: Clone internal kurtosis-cdk repo
+        run: |
+          git clone https://github.com/0xPolygon/kurtosis-cdk.git
+          cd kurtosis-cdk
+          git checkout dan/jit_containers_superusers
+      - name: Install kurtosis
+        run: |
+          echo "deb [trusted=yes] https://apt.fury.io/kurtosis-tech/ /" | sudo tee /etc/apt/sources.list.d/kurtosis.list
+          sudo apt update
+          sudo apt install kurtosis-cli
+      - name: Run kurtosis agent in background
+        run: |
+          kurtosis gateway &  # Run cmd in background
+      - name: Deploy CDK devnet on local github runner
+        run: |
+          cd kurtosis-cdk
+          kurtosis engine restart
```

Comment:
It should also run fine without having to restart the engine (on docker backend).